tests/sentry/integrations/slack/test_unfurl.py

```diff
@@ -60,8 +60,9 @@ def setUp(self):
 
     def test_unfurl_issues(self):
         min_ago = iso_format(before_now(minutes=1))
+        unique_proj = self.create_project()
```

Comment:
You could probably get away with doing:
```py
def setUp() -> None:
  super().setUp()
  self.project = self.create_project()
```
instead.

Reply:
Yeah, this should probably work. Only reason I see it not working is if the multiple test runners share the same test class object.... but that seems really unlikely